updating/updating-clusters-sro.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-sro"]
+= Updating a cluster that includes the SRO
+include::_attributes/common-attributes.adoc[]
+:context: updating-clusters-sro
+
+toc::[]
+
+When updating a cluster that includes the SRO, it is important to consider whether the new kernel module version will be compatible with the kernel modules currently loaded by the SRO. You can run a preflight check to confirm that the SRO will be able to upgrade the kernel modules.
```

Comment:
The same suggestions apply to this line as I provided for the similar line in `updating/index.adoc`.

Reply:
OK